src/main/java/io/aiven/elasticsearch/repositories/security/EncryptionKeyProvider.java

```diff
@@ -43,10 +43,10 @@
     public static final int KEY_SIZE = 256;
 
     public static final Setting<InputStream> PUBLIC_KEY_FILE =
-            SecureSetting.secureFile("gcs.public_key_file", null);
+            SecureSetting.secureFile("encryption.public_key_file", null);
```

Comment:
Maybe, `aiven.gcs` instead?

Reply:
i was thinking about that, but the thing is that we are going to introduce S3 and Azure plugins, and this properties would be the same, since we have same backup keys for the node.

Reply:
gotcha

well, either prefix it with `aiven` or leave as is - whatever suits yo

Reply:
will add `aiven` looks better then encryption ;-)